portal/server/next.config.mjs

```diff
@@ -16,4 +20,37 @@ const nextConfig = {
     },
 }
 
-export default nextConfig
+export default withSentryConfig(nextConfig, {
+// For all available options, see:
+// https://github.com/getsentry/sentry-webpack-plugin#options
+
+org: "mysten-labs",
+project: "walrus-sites-sp-testnet",
+
+// Only print logs for uploading source maps in CI
+silent: !process.env.CI,
+
+// For all available options, see:
+// https://docs.sentry.io/platforms/javascript/guides/nextjs/manual-setup/
+
+// Upload a larger set of source maps for prettier stack traces (increases build time)
+widenClientFileUpload: true,
+
+// Route browser requests to Sentry through a Next.js rewrite to circumvent ad-blockers.
+// This can increase your server load as well as your hosting bill.
+// Note: Check that the configured route will not match with your Next.js middleware, otherwise reporting of client-
+// side errors will fail.
+tunnelRoute: "/monitoring",
```

Comment:
Might need to remove the `tunnelRoute`.

Reply:
question: why remove?

Reply:
Because it might break the routing of the portal. Users might not be able to define a `/monitoring` route in their walrus site.

Reply:
Scratch that, I tried it out and there was no conflict with the WS, meaning that the portal will look up for a /monitoring site resource.